xml/System.Net.Http/ReadOnlyMemoryContent.xml

```diff
@@ -23,7 +23,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides HTTP content based on a ReadOnlyMemory&lt;System.byte&gt;.</summary>
```

Comment:
```suggestion
    <summary>Provides HTTP content based on a <see cref="T:System.ReadOnlyMemory&lt;System.byte&gt;" />.</summary>
```

Reply:
`System.byte` should be `System.Byte` with capital 'B'.